ios/CustomerioReactnative.swift

```diff
@@ -208,6 +209,26 @@ class CustomerioReactnative: NSObject {
         })
     }
     
+    // Tracks `opened` push metrics when a push notification is interacted with.
+    @objc(trackNotificationResponseReceived:)
+    func trackNotificationResponseReceived(payload: NSDictionary) {
+        trackPushMetrics(payload: payload, event: .opened)
+    }
+    
+    // Tracks `delivered` push metrics when a push notification is received.
+    @objc(trackNotificationReceived:)
+    func trackNotificationReceived(payload: NSDictionary) {
+        
+        trackPushMetrics(payload: payload, event: .delivered)
+    }
+    
+    private func trackPushMetrics(payload: NSDictionary, event : Metric) {
+        guard let deliveryId = payload["CIO-Delivery-ID"] as? String, let deviceToken = payload["CIO-Delivery-Token"] as? String else
```

Comment:
Is there any constants in iOS SDK that we can reuse for keys (i.e. `CIO-Delivery-ID` and `CIO-Delivery-Token`)?

Reply:
There wasn't any existing file in native module so I created one and moved all constants to one place. Thanks for the suggestion.